src/utils/network.ts

```diff
@@ -44,14 +44,15 @@ export const queryClient = new QueryClient({
   },
 });
 
-export function buildAntStateQuery({ processId }: { processId: string }): {
+export function buildAntStateQuery({ processId }: { processId?: string }): {
   queryKey: ['ant', string];
   queryFn: () => Promise<AoANTState | null>;
   staleTime: number;
 } {
   return {
-    queryKey: ['ant', processId],
+    queryKey: ['ant', processId ?? 'null'],
```

Comment:
why save - `null` as a string instead of just ''

Reply:
Actually, I should update this to require process id I think